src/Action/Delivery/SuggestOptimizations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Action\Delivery;
+
+use AppBundle\Api\Dto\OptimizationSuggestion;
+use AppBundle\Api\Dto\OptimizationSuggestions;
+use AppBundle\Entity\Delivery;
+use AppBundle\Entity\Task;
+use AppBundle\Service\RouteOptimizer;
+use AppBundle\Service\RoutingInterface;
+use AppBundle\Vroom\RoutingProblem;
+use AppBundle\Vroom\Vehicle;
+
+class SuggestOptimizations
+{
+    public function __construct(private RouteOptimizer $optimizer, private RoutingInterface $routing)
+    {}
+
+    public function __invoke(Delivery $data): OptimizationSuggestions
```

Comment:
why not use `RouteOptimizer.optimizePickupsAndDelivery` ? it seems it does more or less the same thing except storing the gain

Reply:
It doesn't do the same: it's for multiple pickups + one dropoff.

Reply:
OK
maybe create a new method in the RouteOptimizer ? so we have everything there